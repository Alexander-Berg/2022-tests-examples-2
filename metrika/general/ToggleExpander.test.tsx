// import * as React from 'react';
// import { shallow } from 'enzyme';
// import { ReactElementsList } from 'typings/react';
// import { jestSnapshotShallowTest } from 'testing/jest-utils';
// import ToggleExpander from './ToggleExpander';
//
// const componentsForSnapshotRenderTest: ReactElementsList = {
// 'not expanded': <ToggleExpander expanded={false} onClick={jest.fn()} />,
//
// expanded: <ToggleExpander expanded={true} onClick={jest.fn()} />,
// };
//
// describe('ToggleExpander', () => {
// describe('renders', () => {
// jestSnapshotShallowTest(componentsForSnapshotRenderTest);
// });
//
// it('calls onClick callback', () => {
// const onClick = jest.fn();
//
// const component = shallow(
// <ToggleExpander expanded={false} onClick={onClick} />,
// );
//
// component.simulate('click');
// expect(onClick).toHaveBeenCalledTimes(1);
// });
// });
//
