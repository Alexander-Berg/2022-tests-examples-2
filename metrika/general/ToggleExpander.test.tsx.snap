// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToggleExpander renders expanded 1`] = `
<div
  className="ToggleExpander ToggleExpander_expanded"
  onClick={[MockFunction]}
>
  <Bem
    block="ToggleExpander"
    elem="icon"
  >
    <icon />
  </Bem>
</div>
`;

exports[`ToggleExpander renders not expanded 1`] = `
<div
  className="ToggleExpander ToggleExpander"
  onClick={[MockFunction]}
>
  <Bem
    block="ToggleExpander"
    elem="icon"
  >
    <icon />
  </Bem>
</div>
`;
